doc/src/bond_bpm_spring.rst

```diff
@@ -45,9 +48,7 @@ Examples
 Description
 """""""""""
 
-.. versionadded:: 4May2022
```

Comment:
Why are you deleting this?

Reply:
Oops, that was a mistake when I cleaned up a merge conflict. Will revert.

Reply:
You probably want to add a `.. versionadded:: TBD` where you document the new flag(s) (TBD will be replaced with the version data when the release is made). Since people seem to be using old versions of LAMMPS, but reading the online docs, we have started adding these (also retroactively, where easily possible) so that people know whether they can expect a feature to be available for their version.

Reply:
Added those myself.

Reply:
Thank you, I missed that comment. I never noticed the version tags before, so that's good to know for the future.

Reply:
No problem. This is a rather recent thing. Bottom line, I got tired of having to tell people, that tried to use a new feature with an old version of LAMMPS, to look up when changes were made on the bugs and latest features pages in the LAMMPS homepage. I recalled that Python uses this all over the place and decided to start the same with LAMMPS. Mostly with new or changed features and then occasionally also with older features (when convenient).